scripts/test-upgrade.sh

```
@@ -0,0 +1,136 @@
+PROPOSAL_HEIGHT=$(( INITIAL_HEIGHT + 15 ))
+echo "ℹ️  Creating upgrade proposal for version $VERSION and height $PROPOSAL_HEIGHT..."
+cat <<EOF > upgrade-proposal.json
+{
+  "messages": [
+    {
+      "@type": "/cosmos.upgrade.v1beta1.MsgSoftwareUpgrade",
+      "authority": "ethm10d07y265gmmuvt4z0w9aw880jnsr700jpva843",
+      "plan": {
+        "name": "v$VERSION",
+        "height": "$PROPOSAL_HEIGHT",
+        "info": "{\"binaries\":{\"linux/amd64\":\"https://github.com/xrplevm/node/releases/download/v$VERSION/node_$VERSION_Linux_amd64.tar.gz\",\"linux/arm64\":\"https://github.com/xrplevm/node/releases/download/v$VERSION/node_$VERSION_Linux_arm64.tar.gz\",\"darwin/amd64\":\"https://github.com/xrplevm/node/releases/download/v$VERSION/node_$VERSION_Darwin_amd64.tar.gz\",\"darwin/arm64\":\"https://github.com/xrplevm/node/releases/download/v$VERSION/node_$VERSION_Darwin_arm64.tar.gz\"}}"
```

Shellcheck (0.11.0)

[warning] VERSION_Linux_amd64 is referenced but not assigned.

(SC2154)